loleaflet/src/control/Control.Menubar.js

```diff
@@ -1357,6 +1357,13 @@ L.Control.Menubar = L.Control.extend({
 						$(aItem).removeClass(constChecked);
 					}
 				} else if (type === 'action') { // enable all except fullscreen on windows
+					if (id === 'fullscreen') {
+						if (self._map.uiManager.isFullscreen()) {
+							$(aItem).addClass(constChecked);
+						} else {
+							$(aItem).removeClass(constChecked);
+						}
+					}
```

Comment:
if you see there is already a following block for full screen it would be better to reformat and merge the code like this...

```
if (id === 'fullscreen') {
  if (L.Browser.ie || L.Browser.edge) { // Full screen works weirdly on IE 11 and on Edge
    $(aItem).addClass('disabled');
    var index = self.options.allowedViewModeActions.indexOf('fullscreen');
    if (index > 0) {
      self.options.allowedViewModeActions.splice(index, 1);
    }
  } else if (self._map.uiManager.isFullscreen()) {
    $(aItem).addClass(constChecked);
  } else {
    $(aItem).removeClass(constChecked);
  }
}
```